nexa/gguf/streamlit/streamlit_image_chat.py

```diff
@@ -1,29 +1,129 @@
 import os
 import sys
+import subprocess
+import re
+from typing import List, Optional
 from PIL import Image
-from nexa.general import pull_model
 import streamlit as st
+from nexa.general import pull_model
 from nexa.gguf.nexa_inference_image import NexaImageInference
 import io
 
-default_model = sys.argv[1]
-is_local_path = False if sys.argv[2] == "False" else True
-hf = False if sys.argv[3] == "False" else True
+# init:
+DEFAULT_PARAMS = {
+    "num_inference_steps": 50,
+    "height": 512,
+    "width": 512,
+    "guidance_scale": 7.5,
+    "random_seed": 42
+}
 
+def parse_nexa_list_output() -> List[str]:
```

Comment:
why not directly get add a helper funciton here? running "nexa list" with string parsing is not a good practice

https://github.com/NexaAI/nexa-sdk/blob/6b30851fdbe6e263531778e15638a283cd4ed5c1/nexa/general.py#L510-L517